src/main/java/com/google/codeu/servlets/MessageFeedServlet.java

```diff
@@ -26,17 +29,27 @@ public void init(){
     /*Responds with a JSON representation of Message data
       for all users. */
     @Override
-	public void doGet(HttpServletRequest request, HttpServletResponse response)
-		
-	    throws IOException{
-
-	        response.setContentType("application/json");
+	public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException{
+
+    	response.setContentType("application/json");
```

Comment:
I think indentation is off for this line

Reply:
on my end it is 4 spaces... is that not what we are using?

Reply:
Just line 34 looks like an indent is missing